src/main/java/com/ncc/aif/OntToJavaGen/README.md

```diff
@@ -0,0 +1,12 @@
+# Running OntologyGeneration
+
+This class will create Java classes based on the ontology or ontologies used as arguments when running Main.
```

Comment:
Out of curiosity, why is this in a separate README, and in a source code directory?  It seems that this could just be incorporated into a section in the main README.md, maybe after "Running the validator."

Also, I think the README should advise that after generating this file, developers will need to write an ontology-specific subclass of OntologyMapper that uses the constants in the generated file.  They also might want to write ontology-specific tests (e.g., another version of ExamplesAndValidationTests.

Reply:
This class is primarily used for Next Century to update the ontology and add new ontologies whenever changes are made.  While this class is available to use by other users, that is not it's primary purpose.  As such, A segmented README was created in the source code directory for just this class with a link in the top-level directory referencing this README for users who wish to use our Java Generation class.

Reply:
I think the README should advise developers that they'll need to write ontology-specific tests (e.g., another version of ExamplesAndValidationTests), since that's part of the process and presumably we'll do that for the program ontology.  But I'm not hung up on it-- I'm resolving the comment.